src/test/java/com/google/cloud/spanner/pgadapter/nodejs/TypeORMMockServerTest.java

```diff
@@ -502,7 +502,8 @@ public void testCreateAllTypes() throws IOException, InterruptedException {
     assertEquals(
         "some random string", insertRequest.getParams().getFieldsMap().get("p9").getStringValue());
     assertEquals(
-        "{\"key\":\"value\"}", insertRequest.getParams().getFieldsMap().get("p10").getStringValue());
+        "{\"key\":\"value\"}",
+        insertRequest.getParams().getFieldsMap().get("p10").getStringValue());
```

Comment:
Not sure what this change is for?

Reply:
Unrelated, but unfortunately gets forced in in this case: We use an automated code formatter, and it automatically changed this line.